WiseBuy/APIs/APIHelper.m

```diff
@@ -11,7 +11,7 @@
 
 @implementation APIHelper
 
-+ (Item *)createItem: (NSString *)name : (NSString *)description : (NSString *)barcode : (NSString *)imageUrl {
++ (Item *)createItem: (NSString *)name description:(NSString *) description barcode:(NSString *) barcode imageUrl:(NSString *) imageUrl {
```

Comment:
super nit: We usually don't add spaces between the type name and the param name. E.g. 
+ (Item *)createItem:(NSString *)name description:(NSString *)description barcode:(NSString *)barcode imageUrl:(NSString *)imageUrl